learn.html

```diff
@@ -33,7 +35,7 @@
     
     <div id="footer">
       <div class="main">
-        <p>Copyright &copy; 2011 <a href="http://www.nicolas-petton.fr/">Nicolas Petton</a>. The content of this website in licensed under <a href="http://creativecommons.org/licenses/by-nc/3.0/">CC-BY-NC 3.0</a>.</p>
+        <p>Copyright &copy; 2011-2015 <a href="http://www.nicolas-petton.fr/">Nicolas Petton</a>. The content of this website in licensed under <a href="http://creativecommons.org/licenses/by-nc/3.0/">CC-BY-NC 3.0</a>.</p>
```

Comment:
AFAIK the list of years in copyright notice should only include years when substantial (that is, copyrightable) change was made. Maybe `2011, 2015` is more appropriate, then. Just my feeling, though.